paper/src/main/java/vg/civcraft/mc/citadel/CitadelConfigManager.java

```diff
@@ -24,11 +20,14 @@
 import vg.civcraft.mc.civmodcore.dao.ManagedDatasource;
 import vg.civcraft.mc.civmodcore.utilities.TextUtil;
 
+import java.util.*;
```

Comment:
Star imports are generally frowned upon

Reply:
I blame my editor, will fix

Reply:
Fixed in d76977eec8fdeb631f68858548ad4276710c6bdc